core/remotes/resolver.go

```diff
@@ -109,3 +113,62 @@ func WithMediaType(mediatype string) FetchByDigestOpts {
 		return nil
 	}
 }
+
+////
+// Layer fetch config
+////
+
+type FetcherConfig struct {
+	MaxConcurrentDownloads         int
+	MaxConcurrentDownloadsPerLayer int
+	ConcurrentFetchChunksSizeMB    int
+	Limiter                        *semaphore.Weighted
+}
+
+func (cfg FetcherConfig) Parallelism() int64 {
+	// parallel layer fetching remains off when MaxConcurrentDownloadsPerLayer
+	// is set to 0. This could be changed in the future if/when the feature is
+	// successful and widely adopted.
```

Comment:
I think we should change this to `parallel layer fetching remains off if MaxConcurrentDownloadsPerLayer <= 1` because:

1. `MaxConcurrentDownloadsPerLayer=1` equals to no parallel layer fetching.
2. Currently 1 can be returned in 2 places: line 133 if `MaxConcurrentDownloadsPerLayer<= 0` and line 136 if `MaxConcurrentDownloadsPerLayer=1`, which is confusing.

WDYT?

Reply:
That makes more sense, yes, updating.